app/exporters/patients_with_history_exporter.rb

```diff
@@ -93,53 +54,52 @@ def csv_headers
           "BP #{i} Dosage 4",
           "BP #{i} Medication 5",
           "BP #{i} Dosage 5",
-          "BP #{i} Other Medications"]
+          "BP #{i} Other Medications"
+        ]
       end,
       "Latest Blood Sugar Date",
       "Latest Blood Sugar Value",
-      "Latest Blood Sugar Type"].flatten.compact
+      "Latest Blood Sugar Type"
+    ].flatten.compact
   end
 
-  def csv_fields(patient)
-    registration_facility = patient.registration_facility
-    assigned_facility = patient.assigned_facility
+  def csv_fields(patient_summary)
+    patient = patient_summary.patient
     latest_bps = patient.latest_blood_pressures.first(DISPLAY_BLOOD_PRESSURES + 1)
-    latest_blood_sugar = patient.latest_blood_sugar
-    latest_appointment = patient.latest_scheduled_appointment
-    latest_bp_passport = patient.latest_bp_passport
-    fetch_medication_history(patient, latest_bps.map(&:recorded_at))
+    all_medications = fetch_medication_history(patient, latest_bps.map(&:recorded_at))
```

Comment:
This pertains to the refactor (down below) to try to remove the stateful `@medications` out of this class and just pass it around instead.